py_ecc/bls/ciphersuites.py

```diff
@@ -55,7 +55,7 @@ def KeyGen(IKM: bytes) -> Tuple[BLSPubkey, int]:
         l = ceil((1.5 * ceil(log2(curve_order))) / 8)  # noqa: E741
         okm = hkdf_expand(prk, b'', l)
```

Comment:
1. By the spec changes: https://github.com/cfrg/draft-irtf-cfrg-bls-signature/pull/21
> OKM = HKDF-Expand(PRK, key_info || I2OSP(L, 2), L)
2. IMO it's better to note what is `b''` (the `key_info`) here.
3. I think we do need more strict `I2OSP` and `OS2IP` implementation for this library. Reference: https://stackoverflow.com/questions/39964383/implementation-of-i2osp-and-os2ip and https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/blob/8b9c78823e6b691e3e6aabf8116448327094809a/poc/hash_to_field.py#L17-L38

```python
# This parameter MAY be used to derive multiple independent keys from the same IKM. 
# By default, key_info is the empty string.
key_info = b''
okm = hkdf_expand(prk, key_info + I2OSP(l, 2), l)
```